sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10169,6 +10169,70 @@
               "id": "en74evs",
               "defaultValue": ""
             },
+            {
+              "clearOnHide": false,
```

Comment:
I do not recall the impact of this being `false`. I guess that if the component is hidden it will not be part of the payload at all and the only effect would be if the user answers "Yes" to the parent question it will be displayed with the last value populated. Does it make sense?

Reply:
mm, I am wondering why this value is `false` in definition, In formio, its checked, 
![image](https://github.com/bcgov/SIMS/assets/77353155/e4f7f1a7-2b9b-4ab0-b526-f06c1e85ee4e)

Reply:
Got it, this is the panel, for the panel there is no option to `hide on clear`, but I will update the form definition to `true`

Reply:
As discussed, this is for the panel, so Iam not updating it. It has already added for new subquestion.